README.md

```diff
@@ -1,5 +1,5 @@
-# Digital Transfer Guide
-The Rockefeller Archive Center's guide on transfering digital records
+# Guide to Digital Transfer
```

Comment:
So here you've got "Guide to Digital Transfer" and below in the title metadata of `_config.yml` you've got "Guide for Digital Transfer" - I think these need to be the same.